cvs/objects/util/base/include/gcam_data_containers.h

```diff
@@ -141,8 +141,6 @@
 #include "sectors/include/afinal_demand.h"
 #include "sectors/include/ag_supply_sector.h"
 #include "sectors/include/ag_supply_subsector.h"
-#include "sectors/include/capacity_limit_backup_calculator.h"
-#include "sectors/include/CSP_backup_calculator.h"
```

Comment:
do we need these for backwards compatibility here?

Reply:
The model compiled, ran cleanly, and produced expected results without these lines.  But I think you're right that they should be included; I'll reintroduce them in a commit momentarily.